README.md

```diff
@@ -108,4 +108,8 @@ See [`rust/README.md`](rust/README.md)
 
 We use [changesets](https://github.com/changesets/changesets) to release to NPM. You can use the `release` script in `package.json` to publish.
 
+```sh
+yarn release
+```
```

Comment:
The existing tip of using the release script to publish is misleading. Devs should not release manually, they should use merge the "Version Packages" PRs.

Reply:
yup this was a test change I had to trigger the *.md filter - think next time I'll just add an empty `test.md` to make it explicit